### ISSUE TITLE
Frontal appendage

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11591]
**Preferred term label:**
Frontal appendage

**Synonyms**

**Definition (free text, please give PubMed ID)**
A skin projection/s from the fetal forehead as depicted by prenatal ultrasound

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pai syndrome MIM % 155145

**Your nano-attribution (ORCID)**
0000-0001-9947-035X
